### PR TITLE
feat: Add CSS support

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

-   [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What changes did you make? (Give an overview)

- Added support for CSS
- Added a document describing how to add a new language to Code Explorer

### CODE REVIEW

File: src/components/options.tsx [hunk 1]
```diff
@@ -66,6 +66,10 @@ const MarkdownPanel: React.FC = () => {
 	);
 };
 
+const CssPanel: React.FC = () => {
+	return <div />;
+};
+
```
Comment (harish-sethuraman) [inline src/components/options.tsx:72]: You do not have to return an empty div rather you could return null if there were no options

**Comment by nzakas** [comment]
@harish-sethuraman @amareshsm can I get a review on this?

File: docs/adding-languages.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Adding Languages
+
+In order to add a language to Code Explorer, follow these steps.
```
Comment (amareshsm) [inline docs/adding-languages.md:3]:
```suggestion
To add a language to Code Explorer, follow these steps.
```
Can we reframe this? - It is shorter and more direct.

**Review by amareshsm — APPROVED** [review]
LGTM. Keeping it open for a second review.

**Review by harish-sethuraman — APPROVED** [review]

**Pull request merged**